using-valtimo/form-flow/create-form-flow-definition.md

```diff
@@ -0,0 +1,209 @@
+# Creating a form flow definition
+
+A form flow definition is a JSON document that defines steps and which steps can be reached
+under what conditions when the current step is completed. Each step can have zero or more actions
+(e.g. retrieve external data) that trigger when the task is opened or completed. Depending
+on the type of step the step can be handled in a different way. The way a form is handled can
+differ from a payment.
+
+For information on how to link a form flow definition to a task, see 
+[here](../form-link/configure-task.md).
+
+## Definitions
+
+To create form flow definition, the following steps are necessary:
+
+* Create a form flow `.json` file under the following path: `*/resources/config/form-flow/`.
+
+* Add each individual step to the form flow definition, e.g.:
+
+    ```json
+    {
+        "startStep": "start",
+        "steps": [
+            {
+                "key": "start"
+            },
+            {
+                "key": "step2"
+            },
+            {
+                "key": "step3"
+            },
+            {
+                "key": "end"
+            }
+        ]
+    }
+    ```
+
+* Add the next steps for each of the individual steps. This can be done through the `nextStep` property, which
+supports a single step, or `nextSteps`, which supports multiple next steps. Each step should correspond to the key
+of another step defined in this form flow. Since expressions are not yet supported for `nextSteps`, the first step
+will always be selected. So in this example, step 2 will always be followed by step 3.
+
+    ```json
+    {
+        "startStep": "start",
+        "steps": [
+            {
+                "key": "start",
+                "nextStep": "step2"
+            },
+            {
+                "key": "step2",
+                "nextSteps": [
+                    {
+                        "step": "step3"
+                    },
+                    {
+                        "step": "end"
+                    }
+               ]
+            },
+            {
+                "key": "step3",
+                "nextStep": "end"
+            },
+            {
+                "key": "end"
+            }
+        ]
+    }
+    ```
+
+* Configure the step type. Currently, the only step type supported is `form`, which requires a `definition` property to
```

Comment:
Maybe explain more clearly that the properties that need to be set depend on the type and that each type can be different. Also, in the future we will probably need a reference section on the different types and configuration required.

Reply:
Added a reference section and clarified this.